kreuzberg/_extractors/_pdf.py

```diff
@@ -302,25 +423,51 @@ def _extract_pdf_searchable_text_sync(self, path: Path) -> str:
 
     def _extract_pdf_with_ocr_sync(self, path: Path) -> str:
         pdf = None
+        temp_files: list[Path] = []
         try:
-            images = []
             with pypdfium_file_lock(path):
                 pdf = pypdfium2.PdfDocument(str(path))
                 for page in pdf:
-                    bitmap = page.render(scale=200 / 72)
+                    width, height = page.get_size()
+
+                    if self.config.auto_adjust_dpi:
+                        optimal_dpi = calculate_optimal_dpi(
+                            page_width=width,
+                            page_height=height,
+                            target_dpi=self.config.target_dpi,
+                            max_dimension=self.config.max_image_dimension,
+                            min_dpi=self.config.min_dpi,
+                            max_dpi=self.config.max_dpi,
+                        )
+                    else:
+                        optimal_dpi = self.config.target_dpi
+
+                    scale = optimal_dpi / 72.0
```

Comment:
Use a constant fir 72.0, we should replace all places we have it with a single constants from constants.py